src/engraving/libmscore/score.h

```diff
@@ -1438,15 +1446,10 @@ class MasterScore : public Score
     void setTempomap(TempoMap* tm);
 
     bool saveFile(bool generateBackup = true);
-    FileError read1(XmlReader&, bool ignoreVersionError);
```

Comment:
good to see these gone from the public interface!  But it seems sensible to keep at least some support for loading uncompressed mscx files inside libmscore, unless I'm missing something...